openfisca_us/variables/gov/states/md/tax/income/credits/eitc/federal_eitc_without_age_minimum.py

```diff
@@ -9,6 +9,7 @@ class federal_eitc_without_age_minimum(Variable):
     documentation = "The federal EITC with the minimum age condition ignored."
     definition_period = YEAR
     reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-704-effective-until-6302023-for-earned-income"
+    defined_for = StateCode.MD
```

Comment:
Yeah it's an unconventional edge case I didn't think about when writing the original logic. Probably a fix in openfisca-tools (filed in https://github.com/PolicyEngine/openfisca-tools/issues/74) which might get a bit complex: in-built support for simulation-editing formulas in Core would reduce the complexity by a lot. But in the meantime a quick fix is to avoid using `defined_for` for variables whose formulas create a new simulation.